Sources/StreamChatUI/Navigation/ChatThreadListRouter.swift

```diff
@@ -33,3 +33,32 @@ open class ChatThreadListRouter: NavigationRouter<ChatThreadListVC>, ComponentsP
         }
     }
 }
+
+/// A `NavigationRouter` subclass that handles navigation actions of `ChatChannelListVC`.
+@available(iOSApplicationExtension, unavailable)
+open class ChatThreadListStatefulRouter: NavigationRouter<ChatThreadListStatefulVC>, ComponentsProvider {
```

Comment:
Had to create a new router, since the previous one is coupled with `ChatThreadListC`, but this was just to make things work